packages/php/src/HAR/Payload.php

```diff
@@ -334,10 +334,14 @@ protected static function convertHeaderBagToArray(HeaderBag $headers): array
             /** @psalm-suppress PossiblyNullIterator */
             foreach ($values as $value) {
                 // If the header is empty, don't worry about it.
-                if ($value === '') {
+                if ($value === '' || $value === null) {
                     continue; // @codeCoverageIgnore
                 }
 
+                if ($name === 'authorization') {
```

Comment:
is it safe to assume that the header name has been lowercased?

Reply:
Good call out! We might want to use `strtolower()` here.

Reply:
fixed it